src/realtime/realtime.client.ts

```diff
@@ -156,7 +156,7 @@ export class RealtimeClient extends EventEmitter<ToEventFn<RealtimeClientEvents>
                 ? this.emit('disconnect')
                 : this.emitError(new ClientDisconnectedError('MQTToTClient got disconnected.')),
         );
-
+        console.log('Connected to Instagram MQTT')
```

Comment:
This isn't the right place for this. We're not yet connected. There's already a log when the client is connected.